### PR TITLE
CM-83 Implementar login y auth con JWT en users-be

### PR DESCRIPTION
**Pull request by merlindk**

### CODE REVIEW

**Comment by merlindk** [comment]
test this please

**Comment by merlindk** [comment]
test this please

**Comment by merlindk** [comment]
test this please

**Comment by merlindk** [comment]
test this please

**Comment by merlindk** [comment]
@FranciscodeMaussion CR PLS

File: src/main/java/com/cema/users/config/SecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    public AuthenticationManager authenticationManagerBean() throws Exception {
+        return super.authenticationManagerBean();
+    }
+
+    @Bean
+    public FilterRegistrationBean simpleCorsFilter() {
+        UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
+        CorsConfiguration config = new CorsConfiguration();
+        config.setAllowCredentials(true);
+        // *** URL below needs to match the Vue client URL and port ***
```
Comment (FranciscodeMaussion) [inline src/main/java/com/cema/users/config/SecurityConfig.java:65]: I think this have to be applied... maybe as a env var. This way we can have a stronger security for prod and a more flexible in QA

**Review by FranciscodeMaussion — COMMENTED** [review]
this ME disallows the creation of new users, is that on purpouse?

File: src/main/java/com/cema/users/config/SecurityConfig.java (hunk 1 above)
Comment (merlindk) [reply src/main/java/com/cema/users/config/SecurityConfig.java:65]: I want to review security settings later. There are several things wrong for a prod env.

**Review by FranciscodeMaussion — APPROVED** [review]

**Pull request merged**